main/settings.py

```diff
@@ -221,7 +221,10 @@
 LOGIN_ERROR_URL = "/login"
 LOGOUT_URL = "/logout"
 LOGOUT_REDIRECT_URL = "/app"
-OIDC_LOGOUT_URL = get_string("OIDC_LOGOUT_URL", "/logout/oidc")
+MITOL_API_BASE_URL = get_string("MITOL_API_BASE_URL", "")
+OIDC_LOGOUT_URL = get_string(
+    "OIDC_LOGOUT_URL", f"{MITOL_API_BASE_URL.rstrip('/')}/logout/oidc"
```

Comment:
It's better to use `urljoin` here.

Reply:
One thing I was worried about here is the url rewrites - ie `https://api.rc.learn.mit.edu/learn/logout/oidc`

```python
In [4]: urljoin("https://api.rc.learn.mit.edu/learn", "/logout/oidc")
Out[4]: 'https://api.rc.learn.mit.edu/logout/oidc'

In [5]: urljoin("https://api.rc.learn.mit.edu/learn/", "/logout/oidc")
Out[5]: 'https://api.rc.learn.mit.edu/logout/oidc'

In [6]: urljoin("https://api.rc.learn.mit.edu/learn/", "logout/oidc")
Out[6]: 'https://api.rc.learn.mit.edu/learn/logout/oidc'

In [7]: urljoin("https://api.rc.learn.mit.edu/learn", "logout/oidc")
Out[7]: 'https://api.rc.learn.mit.edu/logout/oidc'
```

Reply:
Only the 3rd one above gives the correct result

Reply:
Ok, that makes sense, can you add a code comment on this being a purposeful choice because of this so someone uninformed doesn't make that kind of change and inadvertently break things?

Reply:
thx, added